modules/ROOT/pages/errors/gql-errors/25G02.adoc

```diff
@@ -0,0 +1,11 @@
+= 25G02
+
+== Status description
+error: invalid transaction state - catalog and data statement mixing not supported.
```

Comment:
```suggestion
error: invalid transaction state - catalog and data statement mixing not supported
```

As this is a subcondition without message, I think it should not end with a period right? Same for index page also